src/completers/git.zsh

```diff
@@ -764,6 +803,38 @@ _fzf_complete_git-ls-tree_post() {
     done
 }
 
+_fzf_complete_git-ls-files-and-ls-tree() {
```

Comment:
Generally speaking, a function should represent what it inputs and outputs by its name but not what it does. This function and `_fzf_complete_git-ls-tree` (which I forgot to mention before 😖) is too straightforward in its name as to how it is implemented. For instance, take a look at `_fzf_complete_git-commit-messages`. This function does not indicate its implementation internally utilizing `git log` but is quite obvious over what it does. Consider renaming this function to something like `_fzf_complete_git_files_tree_and_index` (hopefully if I correctly understand this situation 👍 though)